src/utils/percentageChange.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Calculate percentage change between two values
+ * @param current - Current period value
+ * @param previous - Previous period value
+ * @returns Percentage change or null if calculation is not possible
+ */
+export function calculatePercentageChange(current: number, previous: number): number | null {
+  // If signs differ (crossing zero), percentage change is nonsensical
+  if ((current >= 0 && previous < 0) || (current < 0 && previous >= 0)) {
+    return null
+  }
```

Comment:
Important: I don't want to show -1659% which happens happens when net profit swings. I'm opting to _not show any comparison_ when it switches from negative to positive. 
I don't think there's a meaningful % comparison stat to show here.